docs/js/doc-page.js

```diff
@@ -21,6 +21,104 @@ function removeDoubleSlashes(str) {
   return str.replace(/\/{2,}/g, '/');
 }
 
+function getPathFromSlug(str) {
+  return removeDoubleSlashes(`/${str}/`);
+}
+
+function relativeToAbsUrl(url) {
+  if (url.startsWith('http') || url.startsWith('mailto')) {
+    return url;
+  } else
+    return 'https://www.getambassador.io' + removeDoubleSlashes(`/${url}/`);
+}
+
+const getDocPageSchema = ({
+  slug,
+  // an array of { title, slug } relative to parent pages (only if applicable)
+  // This doesn't include the root docs page nor the page itself
+  breadcrumbs = [],
+  // the doc page title (required)
+  title,
+  // the current docs version (required)
+  version,
+  // the current sidebar section (only if applicable)
+  section,
```

Comment:
Do you mean "selection" here?

Reply:
Hm... maybe we've conflicting definitions? What I meant by "section" is where in the sidebar is this link located. If it's nested under a specific "section", such as "Core Concepts" under "Running and Using Ambassador", then its parent should be included in the schema `breadcrumbs`.

Does this make sense?

Reply:
Ah, OK. Yes that makes sense.